nservicebus/operations/tuning_throttling_core_[,5].partial.md

```diff
@@ -1,3 +1,5 @@
+NOTE: By default endpoints are not throttled.
```

Comment:
👍 on the statement, but does it need to be a note?

Reply:
Given that they used to be, at least for some license types, I'd vote to leave it as a note, so it's more visible.